crypto/ec/curve448/arch_ref64/arch_intrinsics.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016 Cryptography Research, Inc.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Originally written by Mike Hamburg
+ */
+
+#ifndef __ARCH_REF64_ARCH_INTRINSICS_H__
+# define __ARCH_REF64_ARCH_INTRINSICS_H__
+
+# define ARCH_WORD_BITS 64
+
+static __inline__ __attribute((always_inline, unused))
```

Comment:
the keywords always_inline, unused etc, can also be written as __always_inline__, __unused__

Reply:
I deleted this file